notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -61,7 +61,7 @@
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        if model_configuration["remote"] is False:
</pre><p>if not model_configuration["remote"]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>